### PR TITLE
get_available_logistic_units will not return None

### PR DESCRIPTION
**Pull request by likecodingloveproblems**
get_available_logistic_units will return an empty QuerySet not None value because we will get error in this corner case other places

### CODE REVIEW

**Comment by likecodingloveproblems** [comment]
@mohsenone I don't test it yet!

**Comment by likecodingloveproblems** [comment]
@mohsenone pls review this if you have time. :pray:

**Comment by likecodingloveproblems** [comment]
I remember you @mohsenone said this is approved. can you pls approve it?

File: logistic/interfaces.py [hunk 1]
```diff
@@ -166,22 +166,21 @@ def _get_free_lu_name(self):
             return lu.name
         return 'ارسال رایگان'
 
+    def shop_active_logistic_units(self, shop):
+        return models.ShopLogisticUnit.filter(
```
Comment (likecodingloveproblems) [inline logistic/interfaces.py:170]:
```suggestion
        return models.ShopLogisticUnit.objects.filter(
```
